samples/java/junit5-selenium/README.md

```diff
@@ -0,0 +1,28 @@
+# JUnit5 sample project
+
+## Pre-requisites for running the tests in the project
+
+- Java must be installed on your system (JDK8 or higher)
```

Comment:
Can we turn this list around? Like:
You need to install the following on your local machine: 
- java
- maven
- etc

Reply:
@d-rede its done now